src/ducks/transactions/TransactionModal.jsx

```diff
@@ -23,11 +31,36 @@ import PropTypes from 'prop-types'
 import flash from 'ducks/flash'
 import { getCategoryId } from 'ducks/categories/helpers'
 import styles from './TransactionModal.styl'
+import iconGraph from 'assets/icons/icon-graph.svg'
+import iconComment from 'assets/icons/actions/icon-comment.svg'
```

Comment:
I think they could be good candidates to include in cozy-ui. Maybe add a todo and open an issue in cozy-ui ?